### PR TITLE
Fixes #3268 updates space in about files to fix synopsis

### LINKED ISSUE #3268: PowerShell about_ help files do not display synopsis due to incompatible formatting
Most powershell core about_* help files are not formatted to support the built in extraction of a synopsis.

Given that the synopsis is the ONLY metadata that can be extracted from the about_ format (which sucks - see #8230 ), these files should be correctly formatted to maximize helpfulness of help using built in features.

Steps to reproduce
------------------

```get-help about_*

```

Expected behavior
-----------------

```about_Scheduled_Jobs              HelpFile                            Describes scheduled jobs and explains how to use and manage
about_Scheduled_Jobs_Advanced     HelpFile                            Explains advanced scheduled job topics, including the file structure
about_Scheduled_Jobs_Basics       HelpFile                            Explains how to create and manage scheduled jobs.

```

Actual behavior
---------------

```about_Try_Catch_Finally           HelpFile
about_Types.ps1xml                HelpFile
about_Type_Operators              HelpFile
about_Updatable_Help              HelpFile
About_Using                       HelpFile

```

Get-Help parses **only the first line** of the SHORT DESCRIPTION into a synopsis.

Most of the about_* files in powershell core are formatted as (**notice blank line after heading**):
```
SHORT DESCRIPTION

    This is the short description
```

for the file synopsis to show up, they should be 

````
SHORT DESCRIPTION
    This is the short description
````

### PR DESCRIPTION
**Pull request by bobbytreed**
Version(s) of document impacted
------------------------------
- [ ] Impacts 7 document
- [X] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[0526a89](https://github.com/bobbytreed/PowerShell-Docs/pull/4329/commits/0526a894eb6855a47fcd3c31ffb746e317f7a2d5)_: 

:clock10: **Full build**: the average full build time is _13 min(s) 28 sec(s)_, this is based on the last _31_ build(s) for this repository.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[0526a89](https://github.com/bobbytreed/PowerShell-Docs/pull/4329/commits/0526a894eb6855a47fcd3c31ffb746e317f7a2d5)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Core/About/about_Operators.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3268b/reference/6/Microsoft.PowerShell.Core/About/about_Operators.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_operators?view=powershell-6&branch=pr-en-us-4329) (powershell-6) | [Details](#user-content-39b59115-0411b522)
[reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_Config.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3268b/reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_Config.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_powershell_config?view=powershell-6&branch=pr-en-us-4329) (powershell-6) | [Details](#user-content-0ccb589e-0411b522)
[reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_Editions.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3268b/reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_Editions.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_powershell_editions?view=powershell-6&branch=pr-en-us-4329) (powershell-6) | [Details](#user-content-171eb305-0411b522)
[reference/6/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3268b/reference/6/Microsoft.PowerShell.Core/About/about_Redirection.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_redirection?view=powershell-6&branch=pr-en-us-4329) (powershell-6) | [Details](#user-content-fe32fb7f-0411b522)
[reference/6/Microsoft.PowerShell.Core/About/about_Registry_Provider.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3268b/reference/6/Microsoft.PowerShell.Core/About/about_Registry_Provider.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_registry_provider?view=powershell-6&branch=pr-en-us-4329) (powershell-6) | [Details](#user-content-7cd8ec5b-0411b522)
[reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3268b/reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_remote_requirements?view=powershell-6&branch=pr-en-us-4329) (powershell-6) | [Details](#user-content-4ccb1e73-0411b522)
[reference/6/Microsoft.PowerShell.Core/About/about_Switch.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3268b/reference/6/Microsoft.PowerShell.Core/About/about_Switch.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_switch?view=powershell-6&branch=pr-en-us-4329) (powershell-6) | [Details](#user-content-7557cc24-0411b522)
[reference/6/Microsoft.PowerShell.Core/About/about_Try_Catch_Finally.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3268b/reference/6/Microsoft.PowerShell.Core/About/about_Try_Catch_Finally.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_try_catch_finally?view=powershell-6&branch=pr-en-us-4329) (powershell-6) | [Details](#user-content-6a27918d-0411b522)
[reference/6/Microsoft.PowerShell.Core/About/about_Variable_Provider.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3268b/reference/6/Microsoft.PowerShell.Core/About/about_Variable_Provider.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_variable_provider?view=powershell-6&branch=pr-en-us-4329) (powershell-6) | [Details](#user-content-5422a97b-0411b522)
[reference/6/Microsoft.PowerShell.Core/About/about_Wildcards.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3268b/reference/6/Microsoft.PowerShell.Core/About/about_Wildcards.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_wildcards?view=powershell-6&branch=pr-en-us-4329) (powershell-6) | [Details](#user-content-b99ab479-0411b522)
[reference/6/Microsoft.PowerShell.Security/About/about_Certificate_Provider.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3268b/reference/6/Microsoft.PowerShell.Security/About/about_Certificate_Provider.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.security/about/about_certificate_provider?view=powershell-6&branch=pr-en-us-4329) (powershell-6) | [Details](#user-content-78430f43-0411b522)
[reference/6/Microsoft.WSMan.Management/About/about_WSMan_Provider.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3268b/reference/6/Microsoft.WSMan.Management/About/about_WSMan_Provider.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.wsman.management/about/about_wsman_provider?view=powershell-6&branch=pr-en-us-4329) (powershell-6) | [Details](#user-content-90ddfbbb-0411b522)
[reference/6/Microsoft.PowerShell.Core/About/about_pwsh.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3268b/reference/6/Microsoft.PowerShell.Core/About/about_pwsh.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_pwsh?view=powershell-6&branch=pr-en-us-4329) (powershell-6) |

<a id="39b59115-0411b522"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Operators.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3268b/reference/6/Microsoft.PowerShell.Core/About/about_Operators.md)
 - **[Warning]** `````Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by Docs platform, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.`````

<a id="0ccb589e-0411b522"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_Config.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3268b/reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_Config.md)
 - **[Warning]** `````Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by Docs platform, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.`````

<a id="171eb305-0411b522"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_Editions.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3268b/reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_Editions.md)
 - **[Warning]** `````Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by Docs platform, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.`````

<a id="fe32fb7f-0411b522"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Redirection.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3268b/reference/6/Microsoft.PowerShell.Core/About/about_Redirection.md)
 - **[Warning]** `````Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by Docs platform, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.`````

<a id="7cd8ec5b-0411b522"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Registry_Provider.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3268b/reference/6/Microsoft.PowerShell.Core/About/about_Registry_Provider.md)
 - **[Warning]** `````Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by Docs platform, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.`````

<a id="4ccb1e73-0411b522"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3268b/reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md)
 - **[Warning]** `````Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by Docs platform, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.`````

<a id="7557cc24-0411b522"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Switch.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3268b/reference/6/Microsoft.PowerShell.Core/About/about_Switch.md)
 - **[Warning]** `````Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by Docs platform, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.`````

<a id="6a27918d-0411b522"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Try_Catch_Finally.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3268b/reference/6/Microsoft.PowerShell.Core/About/about_Try_Catch_Finally.md)
 - **[Warning]** `````Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by Docs platform, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.`````

<a id="5422a97b-0411b522"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Variable_Provider.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3268b/reference/6/Microsoft.PowerShell.Core/About/about_Variable_Provider.md)
 - **[Warning]** `````Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by Docs platform, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.`````

<a id="b99ab479-0411b522"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Wildcards.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3268b/reference/6/Microsoft.PowerShell.Core/About/about_Wildcards.md)
 - **[Warning]** `````Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by Docs platform, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.`````

<a id="78430f43-0411b522"></a>
### [reference/6/Microsoft.PowerShell.Security/About/about_Certificate_Provider.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3268b/reference/6/Microsoft.PowerShell.Security/About/about_Certificate_Provider.md)
 - **[Warning]** `````Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by Docs platform, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.`````

<a id="90ddfbbb-0411b522"></a>
### [reference/6/Microsoft.WSMan.Management/About/about_WSMan_Provider.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3268b/reference/6/Microsoft.WSMan.Management/About/about_WSMan_Provider.md)
 - **[Warning]** `````Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by Docs platform, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C21%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201905211115169345-4329%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=RejCsylxzP6QB8Ky2Oy5XUgLTmbpC2EeZ1xKdzRxbsc%3D&st=2019-05-21T11%3A25%3A47Z&se=2019-06-21T11%3A30%3A47Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_Config.md [hunk 1]
```diff
@@ -401,4 +401,5 @@ This configuration sets a number of options that only work in macOS or Linux:
 
 [About Automatic Variables](./about_Automatic_Variables.md)
 
-[RFC0029]: https://github.com/PowerShell/PowerShell-RFC/blob/master/5-Final/RFC0029-Support-Experimental-Features.md
\ No newline at end of file
+[RFC0029]:
```
Comment (sdwheeler) [inline reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_Config.md]: This needs to be one line. It is a definition for a reference link.

File: reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_Editions.md [hunk 2]
```diff
@@ -240,7 +240,10 @@ to catch possible behavioral differences between editions. For this you must sti
 
 [Pester]: https://github.com/pester/Pester/wiki/Pester
 [PSScriptAnalyzer]: https://github.com/PowerShell/PSScriptAnalyzer
-[`PSUseCompatibleCommands`]: https://github.com/PowerShell/PSScriptAnalyzer/blob/master/RuleDocumentation/UseCompatibleCommands.md
-[`PSUseCompatibleTypes`]: https://github.com/PowerShell/PSScriptAnalyzer/blob/master/RuleDocumentation/UseCompatibleTypes.md
+[`PSUseCompatibleCommands`]:
+https://github.com/PowerShell/PSScriptAnalyzer/blob/master/RuleDocumentation/UseCompatibleCommands.md
```
Comment (sdwheeler) [inline reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_Editions.md]: These are definitions for a reference links. The label and the url need to be on the same line.

File: reference/6/Microsoft.PowerShell.Core/About/about_Variable_Provider.md [hunk 3]
```diff
@@ -109,8 +109,8 @@ drive. To reference an variable from another location, use the drive name
 > PowerShell uses aliases to allow you a familiar way to work with provider
 > paths. Commands such as `dir` and `ls` are now aliases for
 > [Get-ChildItem](../../Microsoft.PowerShell.Management/Get-ChildItem.md),
-> `cd` is an alias for [Set-Location](../../Microsoft.PowerShell.Management/Set-Location.md). and `pwd` is
-> an alias for [Get-Location](../../Microsoft.PowerShell.Management/Get-Location.md).
+> `cd` is an alias for [Set-Location](../../Microsoft.PowerShell.Management/Set-Location.md). and
+> `pwd` is > an alias for [Get-Location](../../Microsoft.PowerShell.Management/Get-Location.md).
```
Comment (sdwheeler) [inline reference/6/Microsoft.PowerShell.Core/About/about_Variable_Provider.md]: Fix the line wrapping.

File: reference/6/Microsoft.PowerShell.Security/About/about_Certificate_Provider.md [hunk 4]
```diff
@@ -92,8 +94,8 @@ Set-Location C:
 > PowerShell uses aliases to allow you a familiar way to work with provider
 > paths. Commands such as `dir` and `ls` are now aliases for
 > [Get-ChildItem](../../Microsoft.PowerShell.Management/Get-ChildItem.md),
-> `cd` is an alias for [Set-Location](../../Microsoft.PowerShell.Management/Set-Location.md). and `pwd` is
-> an alias for [Get-Location](../../Microsoft.PowerShell.Management/Get-Location.md).
+> `cd` is an alias for [Set-Location](../../Microsoft.PowerShell.Management/Set-Location.md). and
```
Comment (sdwheeler) [inline reference/6/Microsoft.PowerShell.Security/About/about_Certificate_Provider.md]: Fix the line wrapping.

File: reference/6/Microsoft.PowerShell.Security/About/about_Certificate_Provider.md [hunk 5]
```diff
@@ -544,7 +546,8 @@ Beginning in Windows PowerShell 3.0, you can get customized help topics for
 provider cmdlets that explain how those cmdlets behave in a file system drive.
 
 To get the help topics that are customized for the file system drive, run a
-[Get-Help](../../Microsoft.PowerShell.Core/Get-Help.md) command in a file system drive or use the `-Path`
+[Get-Help](../../Microsoft.PowerShell.Core/Get-Help.md) command in a file system drive or use the
```
Comment (sdwheeler) [inline reference/6/Microsoft.PowerShell.Security/About/about_Certificate_Provider.md]: Strange line break here.

File: reference/6/Microsoft.WSMan.Management/About/about_WSMan_Provider.md [hunk 6]
```diff
@@ -579,7 +578,8 @@ Beginning in Windows PowerShell 3.0, you can get customized help topics for
 provider cmdlets that explain how those cmdlets behave in a file system drive.
 
 To get the help topics that are customized for the file system drive, run a
-[Get-Help](../../Microsoft.PowerShell.Core/Get-Help.md) command in a file system drive or use the `-Path`
+[Get-Help](../../Microsoft.PowerShell.Core/Get-Help.md) command in a file system drive or use the
+`-Path`
```
Comment (sdwheeler) [inline reference/6/Microsoft.WSMan.Management/About/about_WSMan_Provider.md]: Strange line break here.

**Pull request closed without merging**